agent/testdata/dev/vdc

```diff
@@ -0,0 +1 @@
+����ױ��stub1
```

Comment:
Can you append some extra junk content to one of these files so that we're testing that the length-based stub encoding works?

Reply:
Added that in the `TestParseStubContent`